proms-resources/frontend/src/main/frontend/src/proms/QuestionnaireSet.jsx

```diff
@@ -446,13 +445,12 @@ function QuestionnaireSet(props) {
 
   const getVisitDate = () => {
     let dateAnswer = getVisitInformation("time");
-    return dateAnswer == null ? null : DateQuestionUtilities.amendMoment(DateQuestionUtilities.stripTimeZone(dateAnswer));
+    return DateQuestionUtilities.toPrecision(DateQuestionUtilities.stripTimeZone(dateAnswer));
   }
 
   const appointmentDate = () => {
     let date = getVisitDate();
-    return date == null ? ""
-      : date.formatWithJDF("EEEE, MMMM d, yyyy h:mma");
+    return !date.isValid ? "" : date.toLocaleString(DateTime.DATETIME_MED_WITH_WEEKDAY);
```

Comment:
This causes a JS console error and white screen when the patient signs in without a token and they don't have access to the visit date.
```suggestion
    return !date?.isValid ? "" : date.toLocaleString(DateTime.DATETIME_MED_WITH_WEEKDAY);
```

Reply:
Fixed in da39f3c.